packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -714,3 +748,50 @@ extension ThemeDataExtension on ThemeData {
 
   TextStyle get selectedTextStyle => TextStyle(color: textSelectionColor);
 }
+
+const areaPaneHeaderHeight = 36.0;
+
+/// Create a bordered, fixed-height header area with a title and optional child
+/// on the right-hand side.
+///
+/// This is typically used as a title for a logical area of the screen.
+Widget areaPaneHeader(
```

Comment:
Shouldn't this be a StatelessWidget? That way you don't have to worry about having to pass the BuildContext into it.

Reply:
+1

Reply:
Yes, I agree; I'd tried to make that refactoring in this PR but ran into issues with where these are currently used. The `Split` widget assumes that this returns a `FlexSplitColumnHeader`, and it can call `height` on it. This would need further refactoring to turn it into a statelesswidgt; I'l likely add a todo: to that effect.